spec/features/trainees/creating_a_new_trainee_spec.rb

```diff
@@ -1,13 +1,21 @@
 require "rails_helper"
 
 RSpec.feature "Create trainee journey" do
-  scenario "see summary page afterwards" do
+  scenario "setting up an initial assessment only record " do
     when_i_am_viewing_the_list_of_trainees
-    and_i_click_on_add_data_button
+    and_i_click_on_add_trainee_button
+    and_i_select_assessment_only_route
     and_i_fill_in_train_id
-    and_i_fill_in_first_and_last_name
     and_i_save_the_form
-    then_i_should_see_the_new_trainee_details
+    then_i_should_see_the_new_trainee_overview
+  end
+
+  scenario "setting up an Other record" do
```

Comment:
I wonder if this functionality would be better as a controller unit test. If we're following the sort of 'happy/sad' path thing in our feature specs we wouldn't test every route (as we add them) here. We'd test a successful submission and an error case.

Reply:
done in 4edd2d9